csgo/addons/sourcemod/scripting/ckSurf/misc.sp

```diff
@@ -3136,3 +3136,36 @@ void resetZone(int zoneIndex)
 	g_mapZones[zoneIndex][Team] = 0;
 	g_mapZones[zoneIndex][zoneGroup] = 0;
 }
+
+void SetPlayerVisible(int client)
+{
+	int iAlpha = GetConVarInt(g_hCvarPlayerOpacity);
+	SetEntityOpacity(client, iAlpha);
+
+	// Render weapons opaque too.
+	int iWeapon = -1, iIndex;
+	while((iWeapon = Client_GetNextWeapon(client, iIndex)) != -1)
+		SetEntityOpacity(iWeapon, iAlpha);
+}
+
+void SetPlayerInvisible(int client)
+{
+	SetEntityRenderMode(client, RENDER_NONE);
+}
+
+void SetEntityOpacity(int ent, int iAlpha)
+{
+	if (iAlpha == 255)
```

Comment:
I would personally use iAlpha < 254 rather than = 255

This way it's a bit more safe. do you agree?